docs/goal.md

```diff
@@ -0,0 +1,7 @@
+# What is the goal of Inago?
+Inago is a deployment tool that manages groups of unit files to deploy them to
+a fleet cluster similar to `fleetctl`. Since `fleetctl` is quite limited, Inago
+aims to abstract units away and provide more sugar on top like update
+strategies. That way the user can manage unit files more easily. All actions one
+can apply to a unit using `fleetctl`, can also be applied towards a group using
```

Comment:
> All actions one can apply to a unit using `fleetctl`, can also be applied towards a group using `inagoctl`.

Thats not really true. We have opted to not support `load/unload`, `status` behaves differently (its more like the `list-*`) and `cat` is still to be debated.